File: 30daysofcode/java/Day1-DataTypes.java
import java.util.Scanner;

public class Solution {

    private static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";

        Scanner scan = new Scanner(System.in);

        /* Declare second integer, double and String variables. */
        int j = 0;
        double e = 0.0;
        String t = ""; 
        /* Read and save an integer, doiuble, and String to your variable. */
        j = scan.nextInt();
        e = scan.nextDouble();
        scan.nextLine();
        t = scan.nextLine();
        System.out.println(t);
        /* Print the sum of both integer variables on a new line. */
        System.out.println(i+j);
        /* Print the sum of the doubke variables on a new line. */
        System.out.println(d+e);
        /* Concatenate and print the String variables on a new line;
            the 's' variable above should be printed first */
        System.out.print(s);
        System.out.print(t);

        scan.close();
    }
}

File: 30daysofcode/ccode/Day1-DataTypes.c
#include <stdio.h>

int main() {
    int i = 4;
    double d = 4.0;
    char s[] = "HackerRank ";

   // Declare second integer, double, and String variables.
    int j = 0;
    double e = 0.0;
    char t[100];
    // Read and save an integer, double, and String to your variables.
    scanf("%d", &j);
    scanf("%lf", &e);
    scanf(" %[^\n]", t);
    // Print the sum of both integer varaibles on anew line.
    int intSum = i + j;
    printf("%d\n", intSum);
    // Print the sum of the double variables on a new line.
    double doubleSum = d + e;
    printf("%.1f\n", doubleSum);
    // Concatenate and print the String variables on a new line
    // The 's' variable above should be printed first.
    printf("%s", s);
    printf("%s", t);

    return 0;
}